app/lib/matador/components/errors/error.tsx

```diff
@@ -41,8 +41,11 @@ export const ErrorFallback = ({ error }: ErrorFallbackProps) => {
           footerText={`Matador`}
         >
           <Alert icon={<BiMessageError size={16} />} title="Error" color="red">
-            <Text>Something wrong happened happened!</Text>
-            <Text>{error.status}</Text>
+            {!(error instanceof Error) ? (
+              <Text>Something wrong happened happened: {error.statusText}</Text>
+            ) : (
+              <Text>Sorry but something unexpected happened!</Text>
+            )}
```

Comment:
Is it possible to provide more info about the error?

It would be nice to show `Queue not found` or `job not found` ecc

Reply:
It depends: if you return me the specific error from method we can use the `error.statusText` property or i have to create a  switch with status code and create a text for every case.
Your choice